cpp/src/reductions/std.cu

```diff
@@ -24,8 +24,14 @@
 
 gdf_scalar cudf::reduction::standard_deviation(gdf_column const& col, gdf_dtype const output_dtype, gdf_size_type ddof, cudaStream_t stream)
 {
+#if !defined(__CUDACC_DEBUG__)
```

Comment:
Is it also possible to add CUDA version to the check, considering we'll be fixing it in later CUDA versions. If you're unsure which version it'll be, you can add a TODO.

Reply:
Added "TODO" text